docs/source/checks/nlp/data_integrity/plot_unknown_tokens.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+
+.. _nlp__unknown_tokens:
+
+Unknown Tokens
+**************
+
+This notebook provides an overview for using and understanding the Unknown Tokens check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+==================================
+
+The Unknown Tokens check is designed to help you identify samples that contain tokens unsupported by your tokenizer.
+These unsupported tokens can lead to poor model performance, as the model may not be able to understand the meaning
+of such tokens. By identifying these unknown tokens, you can take appropriate action, such as updating your tokenizer
+or preprocessing your data to handle them.
+
+Generate data & model
+=====================
+
+In this example, we'll use the twitter dataset.
+
+"""
+
+from deepchecks.nlp.datasets.classification import tweet_emotion
+
+dataset, _ = tweet_emotion.load_data()
+
+# %%
+# Run the check
+# =============
+#
+# The check has several key parameters that affect its behavior and output:
+#
+# * `tokenizer`: Tokenizer from the HuggingFace transformers library to use for tokenization. If None,
+#   BertTokenizer.from_pretrained('bert-base-uncased') will be used.
+# * `group_singleton_words`: If True, group all words that appear only once in the data into the "Other" category in
+#   the display.
+
+
+from deepchecks.nlp.checks import UnknownTokens
+
+check = UnknownTokens()
+result = check.run(dataset)
+result.show()
+
+# %%
+# Observe the check's output
+# --------------------------
+#
+# We see in the results that the check found many emojis and some foreign words (Korean) that are not supported by the
```

Comment:
I didn't see in the output any korean words, only emojis...
(if they're grouped then I guess this should be mentioned)

Also (wider comment) -
is there a way (e.g. from the value) to receive all words (and/or ids of the samples in which these words appear? that would be helpful) - which were problematic?

I think this is important to demonstrate it here... as the visualization is nice but not clear if a user now wants to further investigate and fix these issues, how can he do that conveniently and how can he utilize our check's results for that (hoping he can?)